azure-pipelines.yml

```diff
@@ -192,17 +189,16 @@ jobs:
       versionSpec: '$(python.version)'
 
   - bash: |
+      set -e
       sed -i "s/'giotto-tda'/'giotto-tda-nightly'/1" setup.py
       sed -i "s/__version__.*/__version__ = '$(Build.BuildNumber)'/1" gtda/_version.py
       cat gtda/_version.py
-    failOnStderr: true
     condition: eq(variables['nightly_check'], 'true')
     displayName: 'Change name to giotto-tda-nightly'
 
   - task: Bash@3
     inputs:
       filePath: .azure-ci/build_manylinux2010.sh
-      failOnStderr: false
```

Comment:
Looks like pytest failures in the above script were previously ignored, but that still wouldn't explain the identical MacOS failures.

Reply:
Do you regard this as resolved now?